hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -14,7 +14,7 @@ content-type: how-to
 
 # How to target multiple platforms with your WinUI 3 app
 
-Now that you've created your first WinUI 3 app, you might be wondering how to reach more users with a single codebase. This how-to will use Uno Platform to expand the reach of your existing application enabling reuse of the business logic and UI layer across native mobile, web, and desktop.
+Once you've [created](/hub/apps/how-tos/hello-world-winui3.md) a starter Hello World WinUI 3 app, you might be wondering how to reach more users with a single codebase. This how-to will use Uno Platform to expand the reach of your existing application enabling reuse of the business logic and UI layer across native mobile, web, and desktop.
 
 :::image type="content" source="images/hello-world/uno-hello-world-web.png" alt-text="The 'Hello world' app running in the browser.":::
```

Comment:
Can we improve this screenshot to show WinUI target, browser and either android emulator or ios simulator

Reply:
Fixed this, thanks to @agneszitte-nventive !